tests/integration/test_logcollector/test_configuration/test_basic_configuration_log_format.py

```diff
@@ -286,6 +280,18 @@ def check_log_format_invalid(cfg):
                                 error_message=gc.GENERIC_CALLBACK_ERROR_MESSAGE)
 
 
+def check_log_file_duplicated():
+    """Check if Wazuh shows a warning message when the configuration is duplicated."""
+    wazuh_log_monitor = FileMonitor(LOG_FILE_PATH)
+    wazuh_log_monitor.start(timeout=logcollector.LOG_COLLECTOR_GLOBAL_TIMEOUT,
+                            callback=generate_monitoring_callback(
+                                     logcollector.GENERIC_CALLBACK_MSG_LOG_FILE_DUPLICATED),
+                            error_message=logcollector.GENERIC_CALLBACK_ERROR_LOG_FILE_DUPLICATED)
+    wazuh_log_monitor.start(timeout=logcollector.LOG_COLLECTOR_GLOBAL_TIMEOUT,
+                            callback=logcollector.callback_monitoring_macos_logs(),
+                            error_message="The expected macos log monitoring has not been produced")
```

Comment:
Can you define a variable for this message in `logcollector.py` and use it here?

Reply:
Done in commit [eac4f2](https://github.com/wazuh/wazuh-qa/pull/2917/commits/eac4f255eac8a1b641117c24afaa20e11d88d3db)